src/components/Alert.js

```diff
@@ -13,12 +13,15 @@ Alert.defaultProps = {
 
 export default function Alert({ alertMessage }) {
   return (
-    <motion.div animate={{ opacity: 1 }} transition={{ duration: 1 }} initial={{ opacity: 0 }}>
-      <AlertContainer>
-        <img src={check} alt="check-hook" width="16" />
-        <h3>{alertMessage}</h3>
-      </AlertContainer>
-    </motion.div>
+    <AlertContainer
+      as={motion.div}
+      animate={{ opacity: 1 }}
+      transition={{ duration: 1 }}
+      initial={{ opacity: 0 }}
+    >
+      <img src={check} alt="check-hook" width="16" />
```

Comment:
I think if you set `width` as an attribute you should also add `height` as well. Otherwise it is not complete. But not 100% sure.

And I am not sure if you need an `alt` text here of if you could leave it empty. The question is always: Do I need to understand what the image displays in order to understand the content of the page/component? If yes --> descriptive alt text; if no --> leave alt empty.
Here I think you can leave it empty.